03/app.js

```diff
@@ -5,27 +5,57 @@ class Article extends React.Component {
     state = {
         comments: [],
     }
-    
+
+    changeHandler = e => {
+        const {tagName, value} = e.target;
+        this.setState({
+            [tagName.toLowerCase()]: value,
+        })
+    }
+
+    submitHandler = e => {
+        e.preventDefault();
+        const commentsCopy = [...this.state.comments];
+        console.log(this.state.comments)
+        this.addComment(commentsCopy);
+        this.setState({
+            comment: '',
+        })
+    }
+
+    addComment(name) {
+        this.setState({
+            comments: [...this.state.textarea, name],
```

Comment:
Tutaj nalezy wykonać kopie wszystkich komentarzy tj. `comments` i dodać nowy element tj. `content` czyli:
`comments: [...this.state.comments, this.state.content]`